### PR TITLE
Update CHANGES.rst

### PR DESCRIPTION
**Pull request by schymans**
Updated changes since previous release.

### CODE REVIEW

**Comment by jirikuncar** [comment]
Hi @schymans, I guess we need to pin the sympy version.

**Review by jirikuncar — CHANGES_REQUESTED** [review]
Can you pin the sympy version?

**Review by schymans — COMMENTED** [review]
All good, except that we should update the release date, and possibly the version number.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,24 @@
 Changes
 =======
 
+``v0.4.0``
+----------
+*released 2019-11-19*
```
Comment (jirikuncar) [inline CHANGES.rst]:
```suggestion
*released 2019-11-20*
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
 Changes
 =======
 
+``v0.4.0``
```
Comment (jirikuncar) [inline CHANGES.rst]: and let's directly update to `v0.4.1` as `v0.4.0` was already released on Zenodo
```suggestion
``v0.4.1``
```

**Review by jirikuncar — APPROVED** [review]
LGTM

**Pull request merged**